modules/account-lib/test/unit/coin/dot/transactionBuilder/transactionBuilderFactory.ts

```diff
@@ -17,7 +17,7 @@ describe('dot Transaction Builder Factory', function () {
   let factory: TransactionBuilderFactory;
 
   // TODO: BG-43197
-  xdescribe('parse generic builders', function () {
+  describe('parse generic builders', function () {
```

Comment:
@nachofm youre going to have to skip this test again so you can get a green build.

Reply:
Done, drone is passing OK now